src/Milestones/Model.php

```diff
@@ -37,6 +74,21 @@ public function getOutput() {
 		return $output;
 	}
 
+	/**
+	 * Get raw earnings value for Milestone
+	 *
+	 * @return int
+	 * @since 2.9.0
+	 **/
+	protected function getEarnings() {
```

Comment:
I think this is a good candid for refactoring to collections, but that isn't a blocker - just making a note.